spec/models/comment_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+describe Comment, type: :model do
+
+  subject { Comment.new(content: "I'm a comment :)") }
+  
+  describe "Validations" do
+
+    describe '#content' do
+      it 'is invalid when blank' do
+        subject.content = ' ' * 10
+        expect(subject).to_not be_valid
```

Comment:
Will this test pass if you remove content validation? 🤔

Reply:
For some reason, we found that it still passes 🤔
We are investigating if 1. We got the purpose of the test right, 2. We have implemented it correctly

Reply:
So, we found out that removing validation didn't affect the test. 

This was because we were handling creation of the model ourselves like this: `subject { Comment.new(content: "I'm a comment :)") }` instead of allowing Rails to handle this for us.

Changing the above line to `subject { comments(:one) }` fixed this for us 😄

Reply:
This is not entirely true.

The real reason is that before you were creating a comment without user and because user is `belongs_to` association fro comment is is required by rails auto-magically.

When you started to use fixtures with user added to comment, the problem was solved :)

I am going to approve this PR but you might consider making your specs more deterministic by checking also error messages, I mean sth like:

```
subject.valid?
expect(subject.errors.messages.keys.include?(:content)).